README.md

```diff
@@ -1 +1,681 @@
+
+     `UNAUTHORIZED`
+  
+  OR
+
+  * **Code:** 404 RESOURCE NOT FOUND <br />
+    **Content:** <br />
+    
+    ```
+    {'error':'bucket list does not exist'}
+    ```
+
+
+* **Sample Call:**
+
+   * **api/v1/bucketlist/1**<br />
+
+ ----
+
+  **Create bucketlist item**
+  ----
+* **URL**
+
+  `api/<version>/bucketlist/<id>/bucketlistitem/`
+
+* **Method:**
+
+  `POST`
+  
+*  **URL Params**
+
+   **Required:**
+   ```
+   version=[string]
+
+   bucketlist_id=[integer]
+  
+   bucketlist_item_id=[integer]
+   ```
+   
+
+* **Data Params**
+   ```
+   item_name=[string]
+   ```
+
+
+* **Success Response:**
+
+  * **Code:** 201 <br />
+    **Content:**  <br />
+     ```
+     {
+       "bucketlist_item_id": 1,
+       "created_at": "Fri, 31 Mar 2017 08:13:51 -0000",
+       "done": "False",
+       "name": "India",
+       "updated_at": "Fri, 31 Mar 2017 08:13:51 -0000"
+    }
+       ```
+    
+ 
+* **Error Response:**
+
+  * **Code:** 202 ACCEPTED <br />
+    **Content:** <br />
+
+     ```
+     {"message": "item already exists"}
+     ```
+
+
+* **Sample Call:**
+
+   * **api/v1/bucketlist/1/bucketlistitem/**<br />
+
+     **Payload:**
+
+      ```
+      { "item_name": "Taj Mahal"}
+      ```
+   ----
+
+  **Update bucketlist item**
+  ----
+* **URL**
+
+  `api/\<version>/bucketlist/\<id>/bucketlistitem/\<id>`
+
+* **Method:**
+
+  `PUT`
+  
+*  **URL Params**
+
+   **Required:**
+   ```
+   version=[string]
+
+   bucketlist_id=[integer]
+  
+   bucketlist_item_id=[integer]
+   ```
+
+* **Data Params**
+   
+   **Optional:**
+  ```
+   item_name=[string]
+
+   done=[boolean]
+  ```
+
+
+* **Success Response:**
+
+  * **Code:** 201 <br />
+    **Content:**  <br />
+     ```
+     {
+       "bucketlist_item_id": 1,
+       "created_at": "Fri, 31 Mar 2017 08:13:51 -0000",
+       "done": "False",
+       "name": "India",
+       "updated_at": "Fri, 31 Mar 2017 08:13:51 -0000"
+    }
+       ```
+    
+ 
+* **Error Response:**
+
+  * **Code:** 202 ACCEPTED <br />
+    **Content:** <br />
+
+     ```
+     {"message": "item already exists"}
+     ```
+
+
+* **Sample Call:**
+
+   * **api/v1/bucketlist/1/bucketlistitem/1**<br />
+
+     **Payload:**
+
+      ```
+      {"item_name": "Taj Mahal"}
+      ```
+     ----
+
+  **Delete bucketlist item**
+  ----
+* **URL**
+
+  `api/<version>/bucketlist/<id>/bucketlistitem/<id>`
+
+* **Method:**
+
+  `DELETE`
+  
+*  **URL Params**
+
+   **Required:**
+ 
+   ```
+   version=[string]
+
+   bucketlist_id=[integer]
+  
+   bucketlist_item_id=[integer]
+   ```
+
+* **Data Params**
+   
+   None
+
+
+* **Success Response:**
+
+  * **Code:** 201 <br />
+    **Content:**  <br />
+    ```
+    {"message": "bucket list item deleted"}
+    ```
+    
+ 
+* **Error Response:**
+
+  * **Code:** 404 NOT FOUND <br />
+    **Content:** <br />
+
+     ```
+     { "message": "bucket list item does not exist"}
+     ```
+
+
+* **Sample Call:**
+
+   * **api/v1/bucketlist/1/bucketlistitem/1**<br />
+
+ 
```

Comment:
You might want to include a part to explain to the user how the can get your app running locally on their machines.